Moto/Z2PLay/res/values/config.xml

```diff
@@ -0,0 +1,184 @@
+    <string name="config_fusedLocationProviderPackageName" translatable="false">com.qualcomm.location</string>
+    <integer name="config_multiuserMaximumUsers">4</integer>
+    <bool name="config_enableMultiUserUI">true</bool>
+    <bool name="config_batterySdCardAccessibility">false</bool>
+    <string translatable="false" name="config_radio_access_family">GSM | WCDMA | LTE | CDMA | EVDO</string>
+    <bool name="config_device_volte_available">true</bool>
+    <bool name="config_carrier_volte_available">true</bool>
+    <bool name="config_enableWifiDisplay">true</bool>
+   <bool name="config_device_wfc_ims_available">true</bool>
+   <bool name="config_carrier_wfc_ims_available">true</bool>
+    <bool name="config_device_vt_available">true</bool>
+    <string name="config_ims_package">org.codeaurora.ims</string>
+    <bool name="config_dynamic_bind_ims">true</bool>
+    <bool name="config_supportDoubleTapWake">true</bool>
+	<bool name="config_hasTurboPowerCharger">true</bool>
+	<bool name="config_switch_phone_on_voice_reg_state_change">false</bool>
+	<bool name="config_use_sim_language_file">true</bool>
+	<bool name="config_pinnerCameraApp">true</bool>
+	<integer name="config_brightness_ramp_rate_fast">200</integer>
+	<integer name="config_brightness_ramp_rate_slow">40</integer>
+	<integer name="config_screenBrightnessDoze">17</integer>
+	<integer name="config_autoBrightnessBrighteningLightDebounce">2000</integer>
+	<integer name="config_screenBrightnessDark">1</integer>
+        <bool name="config_automatic_brightness_available">true</bool>
+    <integer name="config_screenBrightnessSettingDefault">64</integer>
+   <integer-array name="config_autoBrightnessLevels">
+        <item>1</item>
+        <item>6</item>
+	    <item>12</item>
+        <item>25</item>
+        <item>50</item>
+        <item>80</item>
+        <item>120</item>
+        <item>175</item>
+        <item>250</item>
+        <item>350</item>
+        <item>500</item>
+        <item>750</item>
+        <item>1000</item>
+        <item>1400</item>
+        <item>2000</item>
+        <item>3500</item>
+        <item>6000</item>
+        <item>10000</item>
+        <item>15000</item>
+        <item>20000</item>
+        <item>30000</item>
+        <item>65535</item>
+    </integer-array>
+   <integer-array name="config_autoBrightnessLcdBacklightValues">
+        <item>3</item>
+        <item>7</item>
+        <item>10</item>
+	    <item>13</item>
+        <item>16</item>
+        <item>22</item>
+	    <item>28</item>
+        <item>34</item>
+        <item>43</item>
+        <item>52</item>
+        <item>61</item>
+        <item>72</item>
+        <item>83</item>
+        <item>94</item>
+        <item>105</item>
+        <item>117</item>
+        <item>129</item>
+        <item>141</item>
+        <item>153</item>
+        <item>166</item>
+        <item>179</item>
+        <item>192</item>
+        <item>205</item>
+        <item>218</item>
+        <item>256</item>
```

Comment:
I'm pretty sure 256 doesn't work, it should be 255

Reply:
fix, thanks.